android/app/src/test/java/org/fossasia/openevent/utils/JSONDeserializationTest.java

```diff
@@ -0,0 +1,81 @@
+package org.fossasia.openevent.utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.fossasia.openevent.data.Event;
+import org.fossasia.openevent.data.Microlocation;
+import org.fossasia.openevent.data.Session;
+import org.fossasia.openevent.data.SessionType;
+import org.fossasia.openevent.data.Speaker;
+import org.fossasia.openevent.data.Sponsor;
+import org.fossasia.openevent.data.Track;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+
+import static junit.framework.Assert.assertTrue;
+
+public class JSONDeserializationTest {
+
+    private ObjectMapper objectMapper;
+
+    @Before
+    public void setUp() {
+        objectMapper = new ObjectMapper();
```

Comment:
Instead of this use the same mapper the main code is using and just set the failing on properties as true. This will make sure that the app is working correctly. This is just testing your code implementation